backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/bpartner/service/BPartnerPrintFormatRepository.java

```diff
@@ -44,34 +51,66 @@
 @Repository
 public class BPartnerPrintFormatRepository
 {
-	public BPPrintFormat getByQuery(@NonNull final BPPrintFormatQuery bpPrintFormatQuery)
+	@Nullable
+	public BPPrintFormat getByQuery(@NonNull final BPPrintFormatQuery bpPrintFormatQuery, final boolean onlyCopiesGreaterZero)
```

Comment:
Q: why onlyCopiesGreaterZero is not part of the BPPrintFormatQuery ?

Reply:
thx, fixed